stglib/core/utils.py

```diff
@@ -1405,7 +1405,11 @@ def create_water_level_var(ds):
     Create water level variable from NAVD88 sensor height
     """
 
-    if ds.z.attrs["geopotential_datum_name"] == "NAVD88":
+    if "P_1ac" not in list(ds.data_vars):
+        print("Cannot create water_level variable without P_1ac")
+        return ds
+        exit()
+    elif ds.z.attrs["geopotential_datum_name"] == "NAVD88":
```

Comment:
I don't think this should be an elif, right?

Reply:
I think it has to be elif because as we talked about last time, z could exist but not be relative to NAVD88, so only if it is relative to NAVD88, the water_level will be created.

Reply:
We can discuss this offline, but the first part of the conditional has to do with P_1ac, so I'm still unclear on this.

Reply:
See edit above.  P_1ac and z in NAVD88 are required to create water_level.  There won't be a P_1ac variable if someone ran the code without the atmospheric pressure correction.  But the code still needs to be able to run without the atmospheric pressure correction and not break.